src/talkaction.h

```diff
@@ -101,6 +101,10 @@ class TalkActions final : public BaseEvents
 		bool registerLuaEvent(TalkAction* event);
 		void clear(bool fromLua) override final;
 
+		std::map<std::string, TalkAction> getTalkactions() {
+			return talkActions;
+		}
+
```

Comment:
I find it very confusing that a class named `TalkActions` has a method called `getTalkactions()`. How do you *get* talkactions out of... talkactions?

I would rather add `begin`/`end` to navigate through the internal map instead. At the very least, make this `const&`.